pubspec.yaml

```diff
@@ -5,11 +5,17 @@ description: Replaces Dart script tags with JavaScript script tags,
 author: Seth Ladd <sethladd@gmail.com>
 version: 0.1.0+4
 homepage: https://github.com/sethladd/dart_to_js_script_rewriter
+environment:
+  sdk: '>=1.12.0 <2.0.0'
 dependencies:
-  barback: ">=0.15.0 <0.16.0"
-  html: ">=0.12.0 <0.13.0"
+  barback: '^0.15.2+7'
+  html: '^0.12.2+1'
 dev_dependencies:
-  browser: any
-  test: any
+  browser: '^0.10.0+2'
+  test: '^0.12.6+2'
+  grinder: '^0.8.0+1'
```

Comment:
I use grinder for an easy way to run pub build in the e2e test:
https://github.com/kasperpeulen/dart_to_js_script_rewriter/blob/98956b6d7ee41875458c1248cea5c69bf7e03fbc/test/end2end_test.dart#L2